src/platform/ESP32/nimble/BLEManagerImpl.cpp

```diff
@@ -213,7 +217,7 @@ CHIP_ERROR BLEManagerImpl::_Init()
 {
 #if CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING
 #if CHIP_DEVICE_CONFIG_ENABLE_THREAD
-    if (ConnectivityMgr().IsThreadProvisioned())
+    if (ThreadStackMgrImpl().IsESPThreadProvisioned())
```

Comment:
Why need an ESP specific implementation?

Reply:
We can't use `IsThreadProvisioned()` from  `ConnectivityMrg()` as the`mOTInst` in `GenericThreadStackManagerImpl_OpenThread` is NULL when initializing BLE. But we can not call `ThreadStackMgr().InitThreadStack()` before this check since the `initNetworkCommissioningThreadDriver()` will fail in `ThreadStackMgr().InitThreadStack()`.